### ISSUE TITLE
Guild sdk v2 not returning admins and guildPlatforms

### ISSUE THREAD

**Issue by rulfo71** [guildxyz/guild-sdk#21]
Hey, 

We've been using the API to get a guild with `https://api.guild.xyz/v1/guild/${guildId}` and returns the admins, guildPlatforms and roles from the guild. 

Now i migrated to the v2 sdk  (https://github.com/guildxyz/guild-sdk/blob/main/v2-migration-guide.md) and i see that it doesn't return these fields.
I'm calling ` await guildClient.guild.get(guildId)` and receiving the response i leave below. 

Are this fields going to be added or is it intentionally? 

Thanks

```
{
  id: 11006,
  name: 'Crypto Doberman Club',
  urlName: 'crypto-doberman-club',
  description: '1,666 Dobers on ETH " The Cityzen" \n' + 'Gen 2 of Crypto Doberman Club PFP 2D with 300+ Traits In a future, not too different from our own, when global warming worsened as the years passed, something evolutionary happened to the human race. Humans began to mutate and transform into animals to survive the harsh environments. One of the mutants was inevitably the Doberman species of canines. But they couldn\'t lose the one indispensable symbol of their success: the sign of Crypto.',
  imageUrl: 
    'https://guild-xyz.mypinata.cloud/ipfs/QmP8GFWUhhuD7Sqsm2C9xGLXeZM2t1nDdxQR76Bg9LS28v',
  showMembers: true,
  hideFromExplorer: false,
  socialLinks: {},
  eventSources: {},
  createdAt: '2022-09-02T11:07:12.826Z',
  updatedAt: '2023-03-13T10:57:15.456Z',
  onboardingComplete: true,
  memberCount: 1220,
  guildPin: null,
  theme: {
    mode: 'DARK',
    color: '#C5CFD1',
    backgroundImage: 
      'https://guild-xyz.mypinata.cloud/ipfs/QmWhEspRs6EoArn6B4kmMzCvExCtShfkdywZ8arYTfigCi',
    backgroundCss: null,
    updatedAt: '2023-02-28T12:42:40.901'
  }

}
```

**Comment by cs-balazs** [comment]
Hi @rulfo71, thanks for the report!

`guildPlatforms`, and `roles` are available from separate clients:

```ts

const GUILD_ID = 11006;

const client = createGuildClient("My Project");

const [guild, guildPlatforms, roles] = await Promise.all([
  client.guild.get(GUILD_ID),
  client.guild.reward.getAll(GUILD_ID),
  client.guild.role.getAll(GUILD_ID),
]);

console.log({ ...guild, guildPlatforms, roles });

```

Admins were not included in the initial `2.0.0` release by mistake, there is a PR (https://github.com/guildxyz/guild-sdk/pull/22) for including a separate admin client, which will be usable to retrieve admins like this:

```ts
const admins = await client.guild.admin.getAll(GUILD_ID);
```

### Note on roles
Like guilds, roles have also been split into separate objects. Therefore if you used the `requirements` or the `rolePlatforms` fields of the role objects from the v1 response, you also need to fetch those separately:

```ts
const roles = await Promise.all(
  simpleRoles.map((role) =>
    Promise.all([
      client.guild.role.requirement.getAll(GUILD_ID, role.id),
      client.guild.role.reward.getAll(GUILD_ID, role.id),
    ]).then(([requirements, rolePlatforms]) => ({
      ...role,
      requirements,
      rolePlatforms,
    }))
  )
);
```

**Comment by rulfo71** [comment]
thanks for all the details!

**Comment by rulfo71** [comment]
hey, another field i'm not finding, in `GuildPlatforms` on the api i received the `invite` field which is not on the sdk. 

I see it on the type `PlatformGuildData` though, but missing on the returned data: 

API: 
```
{
      id: 567,
      platformId: 1,
      platformGuildId: '933028279001178182',
      platformGuildData: { inviteChannel: '988742526934589470' },
      platformGuildName: 'ChinChilla Gang',
      invite: 'https://discord.gg/85kH4ByyWn'
    }
```

V2 SDK: 
```
{
      id: 567,
      platformId: 1,
      platformGuildId: '933028279001178182',
      platformGuildData: { inviteChannel: '988742526934589470' }
    }
```

**Comment by cs-balazs** [comment]
@rulfo71 The admin client should now be available in version `2.1.0`
You can find some more info here: https://github.com/guildxyz/guild-sdk#guild-admin-client

Thanks for reporting the missing fields! I've created a separate issue for that: https://github.com/guildxyz/guild-sdk/issues/23

Closing this, as all the data should be accessible in the new version
